src/constants.ts

```diff
@@ -13,6 +13,8 @@ function getBooleanStringVar(variableName: string, defaultValue: boolean) {
   return defaultValue
 }
 
+export const IS_NEW_ROLLOUT = process.env.NEW_ROLLOUT
```

Comment:
It should be `process.env.IS_NEW_ROLLOUT` if the variable is the one you added in the .env.example file

Reply:
The right one was `NEW_ROLLOUT` :)